man/systemd.exec.xml

```diff
@@ -1378,7 +1378,7 @@
               </row>
               <row>
                 <entry>@debug</entry>
-                <entry>Debugging, performance monitoring and tracing functionality (<citerefentry project='man-pages'><refentrytitle>ptrace</refentrytitle><manvolnum>2</manvolnum></citerefentry>, <citerefentry project='man-pages'><refentrytitle>perf_event_open</refentrytitle><manvolnum>2</manvolnum></citerefentry> and related calls)</entry>
+                <entry>Debugging, performance monitoring and tracing functionality (<citerefentry project='man-pages'><refentrytitle>ptrace</refentrytitle><manvolnum>2</manvolnum></citerefentry>, <citerefentry project='man-pages'><refentrytitle>perf_event_open</refentrytitle><manvolnum>2</manvolnum></citerefentry> and related calls). On kernels older than 4.8, it is recommended to restrict this set or at least <citerefentry project='man-pages'><refentrytitle>ptrace</refentrytitle><manvolnum>2</manvolnum></citerefentry> since it can be used to change a system call and escape.</entry>
```

Comment:
@evverx I would say for man pages it would be better to link *official* documentation, and put other links in git logs.

Thanks!